docs/extensibility/display-widgets.md

```diff
@@ -111,7 +111,7 @@ CodeViewer widgets display values using a read-only code editor. The editor auto
 
 ### Table
 
-Table widgets display array data as rows of a table instead of free-standing components. The `children` parameter defines the values used to render the columns. Similar to the `list` section of the Config Map, only inline widgets should be used as children.
+Table widgets display array data as rows of a table instead of free-standing components. The `children` parameter defines the values used to render the columns. Similar to the `list` section of the Config Map, you should use inline widgets only as children.
```

Comment:
```suggestion
Table widgets display array data as rows of a table instead of free-standing components. The **children** parameter defines the values used to render the columns. Similar to the `list` section of the Config Map, you should use inline widgets only as children.
```

Reply:
We use **bold** for parameters, not sure about the section you mention later in the paragraph 🤔